.github/workflows/ci-trilinos.yml

```diff
@@ -95,7 +94,7 @@ jobs:
           -D EIGEN_INCLUDE_DIR=/eigen/eigen-${{ env.eigen_version }} \
           -D Trilinos_DIR=${{ env.trilinos_dir }}/lib/cmake/Trilinos \
           -D CMAKE_INSTALL_PREFIX:PATH=${PRESSIO_INSTALL_DIR} \
-          -D CMAKE_CXX_FLAGS='-Wall -Werror'
+          -D CMAKE_CXX_FLAGS='-Wall'
```

Comment:
This is necessary because Trilinos has deprecated Epetra, resulting in a bunch of warnings:

```
#warning "The Epetra package is deprecated"
```